docs/reference/api.rst

```diff
@@ -0,0 +1,64 @@
+API
+===
+
+SonataClassificationBundle embeds a Controller to provide an API through FOSRestBundle, with its documentation provided by NelmioApiDocBundle.
+
+Setup
+-----
+
+If you wish to use it, you must first follow the installation instructions of both bundles:
+
+* `FOSRestBundle <https://github.com/FriendsOfSymfony/FOSRestBundle>`_
+* `NelmioApiDocBundle <https://github.com/nelmio/NelmioApiDocBundle>`_
+
+Here's the configuration we used, you may adapt it to your needs:
+
+.. code-block:: yaml
+
+    fos_rest:
+        param_fetcher_listener: true
+        body_listener:          true
+        format_listener:        true
+        view:
+            view_response_listener: force
+        body_converter:
+            enabled: true
+            validate: true
+
+    sensio_framework_extra:
+        router:  { annotations: true }
+        request: { converters: true }
+        format_listener:
+            rules:
+                - { path: '^/', priorities: ['json'], fallback_format: json, prefer_extension: false }
+
+    twig:
+        exception_controller: 'FOS\RestBundle\Controller\ExceptionController::showAction'
+
+In order to activate the API's, you'll also need to add this to your routing:
+
+.. code-block:: yaml
+
+    NelmioApiDocBundle:
+        resource: "@NelmioApiDocBundle/Resources/config/routing.yml"
```

Comment:
Please, try to be consistent with the quoting style.
```suggestion
        resource: '@NelmioApiDocBundle/Resources/config/routing.yml'
```

Reply:
In routing we using `"`. Additional it is ignore by DoctorRST (set in [dev-kit](https://github.com/sonata-project/dev-kit/blob/master/config/projects.yaml#L114-L116))